test/parallel/test-domain-uncaught-exception.js

```diff
@@ -88,13 +88,13 @@ function test4() {
   const d5 = domain.create();
   const d6 = domain.create();
 
-  d5.on('error', function onErrorInD2Domain(err) {
+  d5.on('error', function onErrorInD2Domain() {
     process.send('errorHandledByDomain');
   });
 
   d5.run(function() {
     d6.run(function() {
-      setTimeout(function onTimeout() {
+      setImmediate(function onSetImmediate() {
```

Comment:
same here, it does not specify a duration (the second argument), setImmediate is more appropriate.